code/datums/chatmessage.dm

```diff
@@ -26,6 +26,15 @@
 /// The number of z-layer 'slices' usable by the chat message layering
 #define CHAT_LAYER_MAX_Z (CHAT_LAYER_MAX - CHAT_LAYER) / CHAT_LAYER_Z_STEP
 
+/// Approximation of the height
+#define APPROX_HEIGHT(font_size, lines) (font_size * 1.7 * lines) + 2
```

Comment:
you should wrap this in (), so someone sticking * 2 on the end of it gets what they expect